decisions/0003-prefer-open-source-software.md

```diff
@@ -0,0 +1,27 @@
+# 3. Prefer open source software
+
+Date: 2021-04-09
+
+## Status
+
+Accepted
+
+## Context
+
+A question was raised where a system was being developed with Microsoft SQLServer. The solution does not require SQLServer and it would be straightforward to port to [PostgreSQL](https://www.postgresql.org/).
```

Comment:
Does this make it sound like it's not currently using PostgreSQL as a backend?

Reply:
The system in question is currently using SQLServer. The devs are happy to switch as it is easy at this point.